doc/changelog/04-tactics/13403-occs_nums_nat.rst

```diff
@@ -0,0 +1,8 @@
+- **Removed:**
+  "at @occ_nums" clauses in tactics such as autounfold
```

Comment:
I would not use `autounfold` as an example since this tactic is little used. What about using `unfold` instead as an example?

```suggestion
  :n:`at @occ_nums` clauses in tactics such as :tacn:`autounfold`
```